noaa/index.json

```diff
@@ -13,258 +13,426 @@
     },
     "properties": {
       "AWDR": {
-        "type": "keyword"
+        "type": "keyword",
+        "doc_values": {{ doc_values | default(true) | tojson }},
+        "index": {{ index | default(true) | tojson }}
```

Comment:
This is the same space checking change but for noaa which is useful because it is has lots and lots and lots of fields.